opentelemetry-otlp/tests/smoke.rs

```diff
@@ -136,3 +136,107 @@ async fn smoke_tracer() {
     let first_event = first_span.events.first().unwrap();
     assert_eq!("my-test-event", first_event.name);
 }
+
+// Test for partial success response handling
```

Comment:
not sure if we are actually validating anything in this test about partial-success.